src/js/EditController.js

```diff
@@ -0,0 +1,61 @@
+goog.provide('ga_edit_controller');
+
+goog.require('ga_layers_service');
+
+(function() {
+
+  var module = angular.module('ga_edit_controller', [
+    'pascalprecht.translate',
+    'ga_layers_service'
+  ]);
+
+  module.controller('GaEditController', function($scope, $translate,
```

Comment:
Would prefer more specific name.

Reply:
Yes I will change to gaEditGlStyle

Reply:
there is already gaEditGlStyle component